server/http.go

```diff
@@ -35,7 +36,9 @@ func New(
 	longWriter.Log("starting")
 
 	https := &http.Server{
-		Addr: "127.0.0.1:21325",
+		Addr:              "127.0.0.1:21325",
+		ReadHeaderTimeout: 5 * time.Second,
```

Comment:
this is a protection against some DDoS thing... I don't think it applies to us, but I added it anyway